packages-exp/auth-compat-exp/index.rn.ts

```diff
@@ -23,5 +23,9 @@
  */
 
 import { testFxn } from './src';
+import { AsyncStorage } from 'react-native';
```

Comment:
it'd be great if the storage layer were injectable, so if someone were to want to sub in something for AsyncStorage they would be able to do that provided that it conforms to the same interface. redux-persist provides a storage option used on initialization: https://github.com/rt2zz/redux-persist#v6-upgrade

Reply:
That is the overall idea, but this is a polyfill and maintains the legacy behavior.

Reply:
I see that you're big into expo contributions! Is there anything else you think would be helpful from the firebase auth SDK?

Reply:
hey @scottcrossen! @EvanBacon has been doing a bunch of work on authentication apis in expo recently, he may have some questions for you.

can we invite you to our slack to discuss further? if you let me know your email i'd be happy to invite you and other folks on your team so we can collaborate further.

Reply:
Sure sounds good! Email me to get me setup scottcrossen@google.com

Reply:
@scottcrossen - sent!